tests/FakeItEasy.Tests.Approval/ApiApproval.ApproveApiNetStd20.approved.txt

```diff
@@ -7,62 +7,69 @@
 [assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETStandard,Version=v2.0", FrameworkDisplayName="")]
 namespace FakeItEasy
 {
+    [System.Runtime.CompilerServices.NullableAttribute(0)]
     public class static A
     {
         public static FakeItEasy.Configuration.IVoidArgumentValidationConfiguration CallTo(System.Linq.Expressions.Expression<System.Action> callSpecification) { }
         public static FakeItEasy.Configuration.IAnyCallConfigurationWithNoReturnTypeSpecified CallTo(object fake) { }
-        public static FakeItEasy.Configuration.IReturnValueArgumentValidationConfiguration<T> CallTo<T>(System.Linq.Expressions.Expression<System.Func<T>> callSpecification) { }
-        public static FakeItEasy.Configuration.IPropertySetterAnyValueConfiguration<TValue> CallToSet<TValue>(System.Linq.Expressions.Expression<System.Func<TValue>> propertySpecification) { }
-        public static System.Collections.Generic.IList<T> CollectionOfDummy<T>(int numberOfDummies) { }
+        public static FakeItEasy.Configuration.IReturnValueArgumentValidationConfiguration<T> CallTo<[System.Runtime.CompilerServices.NullableAttribute(2)]  T>(System.Linq.Expressions.Expression<System.Func<T>> callSpecification) { }

```

Comment:
the typeparam is marked as "annotated" for some reason, which I guess means... nullable? If I attempt to call the method with a `Expression<Func<string?>>` I get no warning.
Confirmed! If I constrain by `notnull`, I get a warning in my call with an `Expression<Func<string?>>`.